core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -156,4 +156,11 @@ private[spark] object ThreadUtils {
         result
     }
   }
+
+  /**
+   * Construct a new ForkJoinPool with a specified max parallelism.
+   */
+  def newForkJoinPool(maxThreadNumber: Int): scala.concurrent.forkjoin.ForkJoinPool = {
```

Comment:
Do we need a method for this, really, vs just calling that constructor?
can it be imported as a class name?